### PR TITLE
[ADD] Added functions and models for Shipping Label and FBB Endpoints

### PR DESCRIPTION
**Pull request by bharti-dreambits**

### CODE REVIEW

File: bol/retailer/api.py [hunk 1]
```diff
@@ -299,6 +327,67 @@ def handleReturnItem(self, rmaId, status_reason, qty):
...
+
+    def get(self, **params):
+        response = self.request("GET", params=params)
+        return Replenishments.parse(self.api, response.text)
+
+    def create(self, params):
+        response = self.request("POST", json=params)
+        return ProcessStatus.parse(self.api, response.text)
+
+    def pickupTimeSlots(self,address, numberOfLoadCarriers):
```
Comment (karan-dreambits) [inline bol/retailer/api.py:342]: Is this method used in the current flow we are going to work on with Odoo ?

File: bol/retailer/api.py [hunk 2]
```diff
@@ -299,6 +327,67 @@ def handleReturnItem(self, rmaId, status_reason, qty):
...
+        params = {
+            "address" : address,
+            "numberOfLoadCarriers" : numberOfLoadCarriers
+        }
+        response = self.request("POST", path="pickup-time-slots", json=params)
+        return TimeSlots.parse(self.api, response.text)
+
+    def productLabels(self, labelFormat, products):
```
Comment (karan-dreambits) [inline bol/retailer/api.py:350]: what is the purpose of this function ? Either it gets or posts/creates a productLabel, right ? 
If so then the name of function is ambiguous. 
ideally it should be getProductLabels

File: bol/retailer/api.py [hunk 3]
```diff
@@ -299,6 +327,67 @@ def handleReturnItem(self, rmaId, status_reason, qty):
...
+
+    def getById(self, replenishment_id):
+        response = self.request("GET", path=str(replenishment_id))
+        return Replenishment.parse(self.api, response.text)
+
+    def update(self, replenishment_id, **param):
+        response = self.request("PUT", path=str(replenishment_id), json=param)
+        return ProcessStatus.parse(self.api, response.text)
+
+    def loadCarrierLabels(self, replenishment_id, label_type="WAREHOUSE"):
```
Comment (karan-dreambits) [inline bol/retailer/api.py:370]: same as above. Function name is ambiguous

**Review by karan-dreambits — CHANGES_REQUESTED** [review]

File: bol/retailer/api.py (hunk 2 above)
Comment (bharti-dreambits) [reply bol/retailer/api.py:350]: Ok sir will change the name

File: bol/retailer/api.py (hunk 3 above)
Comment (bharti-dreambits) [reply bol/retailer/api.py:370]: Will change the function name to more meaningful

File: bol/retailer/api.py (hunk 1 above)
Comment (karan-dreambits) [reply bol/retailer/api.py:342]: so this is not going to be used in the current worklow, right ? 
What is the purpose of this endpoint/feature ?

**Review by karan-dreambits — APPROVED** [review]

**Pull request merged**